views/topic/topic.hbs

```diff
@@ -1,4 +1,4 @@
-<script src="/vendor/geoGebra/deployggb.js"></script>
+<script src="https://www.geogebra.org/apps/deployggb.js"></script>
```

Comment:
@janrenz was sagst du zu externen domains?

Reply:
in dem sktipt wird eh alles andere von geogebra geladen @janrenz @adrianjost 
damit haben wir dann auch immer das aktuelle skript - alle skripte gibt es nicht lokal.

Reply:
Okay